lib/iron_bank/describe/excluded_fields.rb

```diff
@@ -87,9 +88,65 @@ def valid_query?
         info "Successful query for #{object_name}"
 
         true
-      rescue IronBank::InternalServerError, IronBank::BadRequestError => e
+      rescue IronBank::BadRequestError => e
         @last_failed_fields = extract_fields_from_exception(e)
 
+        false
+      rescue IronBank::InternalServerError
+        @last_failed_fields = exctract_from_dividing
+
+        false
+      end
+
+      def exctract_from_dividing
```

Comment:
this approach while doing a binary search could still be expensive since we'd be repeatedly querying -- let's find out and document the places where `IronBank::Describe::ExcludedFields.call(...)` is being invoked.

Reply:
Could it be enough? I found all places where both `IronBank::Describe::ExcludedFields` or `IronBank::Schema.excluded_fields` are called

https://github.com/zendesk/iron_bank/pull/107/commits/ba6b6a2f1926b4a1538a56924d9cc7dc76ae9d0b
https://github.com/zendesk/iron_bank/pull/107/commits/e3b606afd24abfb0afa0ca00aa75ca33c2ad2aa9
https://github.com/zendesk/billing/pull/16349